Signal/src/util/OWSWindowManager.m

```diff
@@ -473,7 +432,7 @@ - (void)ensureScreenBlockWindowShown
     }
 
     self.screenBlockingWindow.windowLevel = UIWindowLevel_ScreenBlocking();
-    [self.screenBlockingWindow.rootViewController becomeFirstResponder];
+    [self.screenBlockingWindow makeKeyAndVisible];
```

Comment:
Hmm, this is a significant change.  I've wondered whether what the exact difference is between `makeKeyAndVisible` and simply manipulating the window's visibility.  

I had something like this in an early iteration of the window manager and I moved away from it because I found an issue.  I'm not against this, but we need a rigorous test plan that includes all permutations of:

* Screen lock enabled or disabled.
* Call View presented or not.
* Input toolbar visible or not.
* Input toolbar selected or not.
* Another view presented over conversation view using `present`.
* Another view presented over conversation view using `navigation controller`.

Note: To my surprise I found that it's necessary to repeat tests around responder state and input toolbar visibility _twice_ because in some cases issue occur only the second time.